test/clj/matcher_combinators/matchers_test.clj

```diff
@@ -361,6 +361,23 @@
                               :d (m/embeds {:e {:inner-e {:x 1 :y 2}}})}})
            actual)))))
 
+(deftest strictly-equals-matcher
+  (testing "passing case with one level map"
+    (is (match? (m/strictly-equals {:a :b})
+                {:a :b})))
+
+  (testing "passing case with multi level map"
+    (is (match? (m/strictly-equals {:a :b :c {:d {:e :f}}})
+                {:a :b :c {:d {:e :f}}})))
+
+  (testing "failing case with one level map"
+    (is (no-match? (m/strictly-equals {:a :b})
+                   {:a :b :c :d})))
+
+  (testing "failing case with multi level map"
+    (is (no-match? (m/strictly-equals {:a :b :c {:d {:e :f}}})
+                   {:a :b :c {:d :e}}))))
```

Comment:
this would also fail without the `strictly-equals`
Maybe something like 
```clojure
(is (match? (m/strictly-equals {:a :b :c {:d :e}})
            {:a :b :c {:d :e
                       :y :z}}))
```